sdb/command.py

```diff
@@ -127,8 +122,14 @@ def call(self,
         # pylint: disable=missing-docstring
         raise NotImplementedError
 
+    def _call_and_yield(self,
+                        objs: Iterable[drgn.Object]) -> Iterable[drgn.Object]:
+        result = self.call(objs)
+        if result is not None:
```

Comment:
If the result is `None`, do we still return an `Iterable[drgn.Object]` (as the function signature suggests)?

Reply:
Yes, because this is a generator function, which always returns an Iterator, which is the generator.